projects/components/CHANGELOG.MD

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.0.0-dev.1]
+### Fixed
+- Showing grouped actions with no children in the quick search as a search result. Eg., If a vapp action called
```

Comment:
This description should not be that long. It should be clear enough to find the corresponding commit, ideally it should repeate the first line of the commit message